dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/ws/security/kerberos/internal/KerberosExtServiceImpl.java

```diff
@@ -63,28 +70,10 @@
     @Reference(policy = ReferencePolicy.STATIC, target = "(java.specification.version>=1.8)")
```

Comment:
Is this version check still needed?

Reply:
Every Java version we support is >= 1.8 so it seems unnecessary to have that target like Tom said.

Reply:
It would seem that setProcess isn't needed any longer with us supporting Java 8 and higher for Liberty based off of the comment.

Reply:
We compile to Java 8 byte code.  None of the liberty code can run with a Java before Java 8.  That is why this seems unnecessary.